R/props.R

```diff
@@ -1,14 +1,19 @@
 #' Check Proportions
-#' 
-#' Checks if x is proportions vector - non-missing dbls between 0 and 1 that sum to 1.
+#'
+#' Checks if x is proportions vector - non-missing dbls between 0 and 1 that sum
+#' to 1.
 #'
 #' @inheritParams params
-#' 
+#'
 #' @return An invisible copy of x (if it doesn't throw an error).
 #' @export
 check_props <- function(x, x_name = substitute(x),
                        error = TRUE) {
-  lifecycle::deprecate_soft("0.5.1", "check_props()", "chk::check_values()")
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_props()", 
+    NULL,
+    "Please use chk::check_values() with values = c(0, 1) instead")
```

Comment:
Can you change to

lifecycle::deprecate_soft(
     "0.5.1", 
     "check_props()", 
     chk::chk_range())

Reply:
Love this combo, we are getting at both the sum and the range of values. Great idea @nadinehussein